ninchatsdk/src/main/java/com/ninchat/sdk/activities/NinchatChatActivity.java

```diff
@@ -568,14 +569,17 @@ public void setRootViewHeightListener() {
             // Keyboard hidden
             if (heightDiff > 0 && getResources().getConfiguration().orientation == Configuration.ORIENTATION_PORTRAIT) {
                 layoutParams.height = (int) getResources().getDimension(R.dimen.ninchat_chat_activity_video_view_height);
+                // Update video height and cache current rootview height
+                videoContainer.setLayoutParams(layoutParams);
             }
             // Keyboard visible
             else if (heightDiff < 0 && getResources().getConfiguration().orientation == Configuration.ORIENTATION_PORTRAIT) {
                 layoutParams.height = (int) getResources().getDimension(R.dimen.ninchat_chat_activity_video_view_height_small);
+                // Update video height and cache current rootview height
+                videoContainer.setLayoutParams(layoutParams);
+                // push messages on top of soft keyboard
+                messageAdapter.scrollToBottom(false);
             }
-
-            // Update video height and cache current rootview height
-            videoContainer.setLayoutParams(layoutParams);
```

Comment:
I removed this line from here and moved it inside condition scope. The reason for that, `setRootViewHeightListener` call very frequently and it is not optimal to call at this rate even though the layout parameters are not changing. So, I thought we only want to set layout when they are updated. Please, let me know if there were other reasons for setting layout params outside if, else scope that I am missing.

cc @Toni-

Reply:
The changes are fine, no particular reason for this to be here